awx/main/utils/handlers.py

```diff
@@ -294,6 +294,18 @@ def __init__(self, **kwargs):
         super(AWXProxyHandler, self).__init__(**kwargs)
         self._handler = None
         self._old_kwargs = {}
+        self._auditor = logging.handlers.RotatingFileHandler(
+            filename='/var/log/tower/external.log',
+            maxBytes=1024 * 1024 * 5, # 5 MB
+            backupCount=5,
+        )
```

Comment:
this is probably laughably small if job events are in it

Reply:
This probably needs perf tested at scale in general.

Reply:
Maybe?  The intention isn't to turn this on log-term; just long enough to see if there are missing events.  Maybe 50MB or 100MB would be better?

Reply:
If the intention is as a debugging aid, is there a way to do this that doesn't involve code modification?

I worry that customers will decide This Is The Way

Reply:
Not the doesn't involve bespoke `/etc/tower/settings.py` monkey business across all your nodes.